src/internal/plugins/controllers/drawers.ts

```diff
@@ -15,16 +16,10 @@ export type DrawersRegistrationListener = (drawers: Array<DrawerConfig>) => void
 export class DrawersController {
   private drawers: Array<DrawerConfig> = [];
   private drawersRegistrationListener: DrawersRegistrationListener | null = null;
-  private updateTimeout: ReturnType<typeof setTimeout> | null = null;
 
-  private scheduleUpdate() {
-    if (this.updateTimeout) {
-      clearTimeout(this.updateTimeout);
-    }
-    this.updateTimeout = setTimeout(() => {
```

Comment:
I have figured that this timeout is basically a debounce implementation and I can reuse our existing one